pkg/convert/xliff12.go

```diff
@@ -64,19 +64,24 @@ func FromXliff12(data []byte, original bool) (model.Messages, error) {
 	}
 
 	getMessage := func(t transUnit) string { return t.Source }
+	status := model.MessageStatusTranslated
 
 	// Check if a target language is set
 	if !messages.Original {
```

Comment:
Can you please replace the negation with the following? It should improve readability.

```suggestion
	if messages.Original {
```